openpype/hosts/fusion/plugins/load/load_sequence.py

```diff
@@ -129,6 +132,9 @@ class FusionLoadSequence(load.LoaderPlugin):
 
     families = ["imagesequence", "review", "render", "plate"]
     representations = ["*"]
+    extensions = set(
+        ext.lstrip(".") for ext in IMAGE_EXTENSIONS
+    )
```

Comment:
Even though not entirely stable the Fusion Loaders do support video formats to as far as I know. We rarely use that however so I'm not sure it's needed here. Just wanted to mention it here.

@EmberLightVFX what do you think?

Reply:
I could see only allowing to load sequences as a problem. 

We sometimes work with r3d files directly.
Our stock footage library contains a lot of video files that we haven't converted to exr yet, and honestly probably won't.

Reply:
I see, I was looking into the code and there was not mention of video files, also the fine name suggested image sequences. I can of course add video extension support too.